website/docs/r/virtual_machine_scale_set.html.markdown

```diff
@@ -346,6 +346,27 @@ The following arguments are supported:
 
 `storage_profile_image_reference` supports the following:
 
+* `id` - (Optional) Specifies the ID of the (custom) image to use to create the virtual 
+machine scale set, for example:
```

Comment:
Thanks for making those changes - I was thinking more of linking to an example within the Examples directory on Github - but on reflection perhaps that's a future task once this has been merged, and this is fine for the moment?